CHANGELOG.md

```diff
@@ -6,6 +6,25 @@
 ### Removed 
 ### Security
 
+# 2019-03-05-1
+## Portal
+### Added
+- Added a setup script
+### Changed
+- Show logo and header colour as per current deployment env config
+  - Change logo and banner (header) colour of simple server as per SIMPLE_SERVER_ENV
+  - Added deployment env string to page title
+### Fixed
+- Correctly display BP counts for users who switch facilities
+- Hide BPs with orphaned User associations
```

Comment:
Wasn't this bug introduced in the `Correctly display BP counts` fix?
If so, doesn't need to be mentioned if fixed in the same PR.